package/MDAnalysis/coordinates/memory.py

```diff
@@ -167,7 +161,7 @@
 
   u2 = mda.Merge(protein).load_new(
            AnalysisFromFunction(lambda ag: ag.positions.copy(),
-                                protein).run().results.swapaxes(0, 1),
```

Comment:
It's really nice that the swapaxes are gone now. That makes the code easier to understand